doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -106,15 +102,13 @@ The architecture must support a way to isolate the compute resources of the infr
+As all production networking solutions for Kubernetes are based on CNI plugins and the implicit requirement in [4.2.2 Virtual Network Interface Specifications](../../../ref_model/chapters/chapter04.md#422-virtual-network-interface-specifications) that documents the requirement to have the capability to attach several network interfaces to the pods, the CNTT architecture must support a CNI metaplugin/CNI multiplexer.
 
-A CNI metaplugin/CNI multiplexer is capable to attach several interfaces, using different other CNI plugins, to a pod. The different network characteristics of the interfaces, which require different networking technologies require different CNI plugins.
+A CNI metaplugin/CNI multiplexer has the capability to attach several interfaces, using different other CNI plugins, to a pod. Note that the different network characteristics of the interfaces might require different networking technologies, which would potentially require different CNI plugins.
 
-To comply with `req.inf.ntw.08` inter node communication must be served by a CNI plugin which complies with the default K8s networking assumptions.
+To comply with `req.inf.ntw.08`, inter node communication must be served by a CNI plugin which complies with the default K8s networking assumptions.
 
-There are two types of low latency and high throughput networks required by telecom workloads. Network used for signalling traffic are more demanding than what an overlay network can handle, but still does not need the usage of user space networking. Due to the nature of signalling protocols used, this type of networks require the NAT-less communication stated by `infra.net.cfg.003`. Due to the combination of these two requirements networks with these characteristics must be served by a CNI plugin with IPVLAN or MACVLAN support.
-
-The low latency, high throughput networks for handling the user plane traffic require the capability to use an user space networking technology.
+There are two types of low latency and high throughput networks required by telco workloads: Signalling traffic workloads and user plane traffic workloads. Networks used for signalling traffic are more demanding than what a standard overlay network can handle, but still do not need the use of user space networking. Due to the nature of the signalling protocols used, this type of networks require the NAT-less communication documented in `infra.net.cfg.003`. Due to the combination of these two requirements networks with these characteristics must be served by a CNI plugin with IPVLAN or MACVLAN support. On the other hand, the low latency, high throughput networks used for handling the user plane traffic require the capability to use a user space networking technology.
```

Comment:
```suggestion
There are two types of low latency and high throughput networks required by telco workloads: Signalling traffic workloads and user plane traffic workloads. Networks used for signalling traffic are more demanding than what a standard overlay network can handle, but still do not need the use of user space networking. Due to the nature of the signalling protocols used, these type of networks require NAT-less communication documented in `infra.net.cfg.003`. Due to the combination of these two requirements, networks with these characteristics must be served by a CNI plugin with IPVLAN or MACVLAN support. On the other hand, the low latency, high throughput networks used for handling the user plane traffic require the capability to use a user space networking technology.
```

Reply:
@bfcohen @tomkivlin "Due to the combination of these two requirements, ..." which two requirements?

Reply:
Good question - let me check the commits on that page, it may have been there were two requirements listed previously.

Reply:
The original wording was:
> There are two types of low latency and high throughput networks required by `req.inf.ntw.04`. Network used for signalling traffic <_removed_> require the NAT-less communication stated by `req.inf.ntw.03`. Due to the combination of these two requirements networks with these characteristics must be served by a CNI plugin with IPVLAN or MACVLAN support.

So the two requirements were listed.  

I would suggest modifying the line to:
> There are two types of low latency and high throughput networks required by telco workloads: signalling traffic workloads and user plane traffic workloads. Networks used for signalling traffic are more demanding than what a standard overlay network can handle, but still do not need the use of user space networking. Due to the nature of the signalling protocols used, these type of networks require NAT-less communication documented in `infra.net.cfg.003` and will need to be served by a CNI plugin with IPVLAN or MACVLAN support. On the other hand, the low latency, high throughput networks used for handling the user plane traffic require the capability to use a user space networking technology.

Reply:
Original wording, for completeness: https://github.com/cntt-n/CNTT/commit/686ebb92c37272033b616cc1c00675986cb62c10#diff-44f0d56ff3bb53bd7d3d1facd9e902ffL115